src/sandbox.rs

```diff
@@ -71,7 +71,7 @@ pub(crate) fn from_config(config: &SandboxConfig) -> Result<Option<Box<dyn Sandb
     // of it because it might contain crates.io credentials, which we'd like to avoid exposing.
     let cargo_home = &home.join(".cargo");
     sandbox.ro_bind(&cargo_home.join("bin"));
-    sandbox.ro_bind(&cargo_home.join("git"));
+    // sandbox.ro_bind(&cargo_home.join("git")); ?
```

Comment:
I'm guessing cargo only creates this directory when you have dependencies from git. I've pushed a change so that ro_bind now checks if the directory exists before it binds it, so it should be fine to leave this line in now

Reply:
Cool cool. I just re-merged master and reverted that hack. CI now compiles. I left some items as TODO later.